Canvas/TechDebt/Files and Folders/FileViewController.h

```diff
@@ -43,5 +44,9 @@
 @property (nonatomic, copy) NSString* pageViewEventName;
 
 - (void)showDownloadError:(NSError *)error;
+- (BOOL)previewController:(QLPreviewController *)controller shouldOpenURL:(NSURL *)url forPreviewItem:(id<QLPreviewItem>)item;
+- (NSInteger)numberOfPreviewItemsInPreviewController:(QLPreviewController *)controller;
+- (id<QLPreviewItem>)previewController:(QLPreviewController *)controller
+                    previewItemAtIndex:(NSInteger)index;
```

Comment:
I can try removing it. I think I added it because I was getting an error saying that the method wasn't defined, but it's possible that was something else. I will try removing these and see what happens.